### ISSUE TITLE
Need to add the variable `mbuffer_block_size`

### ISSUE THREAD

**Issue by click0** [jimsalterjrs/sanoid#519]
Need to add the variable `mbuffer_block_size`.
Because for large ZFS strippers with large `recordize` files, you can and should set more than 128K. For example, I have a value of 1M.

```
my $mbuffer_block_size = "128K";
...
GetOptions(\%args, "no-command-checks", "monitor-version", "compress=s", "dumpsnaps", "recursive|r", "sendoptions=s", "recvoptions=s",
                   "source-bwlimit=s", "target-bwlimit=s", "sshkey=s", "sshport=i", "sshcipher|c=s", "sshoption|o=s@",
                   "debug", "quiet", "no-stream", "no-sync-snap", "no-resume", "exclude=s@", "skip-parent", "identifier=s",
                   "no-clone-handling", "no-privilege-elevation", "force-delete", "no-clone-rollback", "no-rollback",
                   "create-bookmark", "pv-options=s" => \$pvoptions,
                   "mbuffer-block-size=s" => \$mbuffer_block_size,
                   "mbuffer-size=s" => \$mbuffer_size) or pod2usage(2);
...
my $mbufferoptions = "-q -s $mbuffer_block_size -m $mbuffer_size 2>/dev/null";

```

**Comment by phreaker0** [comment]
I don't see how this will change anything. Did you check if this has any performance advantage? muffer reads the data from a pipe and to a pipe, in most cases the data is also sent over the network where it's broken up into smaller frames anyway.

**Comment by click0** [comment]
Yes, here you need to test the behavior of mbuffer on different OS ....

**Comment by click0** [comment]
I still have the initial synchronization of the partition with large files in total ~ 10TB.
12 hours ago I changed `mbuffer_block_size` to 2 M.

I will attach graphs with monitoring.

![graph_image php-iops](https://user-images.githubusercontent.com/396824/76089386-3b375e80-5fc2-11ea-83b7-c76a52c4b34d.png)

![graph_image php-iops-2](https://user-images.githubusercontent.com/396824/76089394-4094a900-5fc2-11ea-8ccc-65593d19663c.png)

The load schedule directly on the raw HDD has not changed.
![graph_image php-iops3-raw](https://user-images.githubusercontent.com/396824/76089699-d7f9fc00-5fc2-11ea-969f-debd1a4aabde.png)
